### ISSUE TITLE
Docker login with deployment service token fails

### ISSUE THREAD

**Issue by ashb** [astronomer/houston-api#104]
Trying to use a service token to log in to the registry fails:

```
2019-08-14T15:18:33 INFO Processing registry authorization
(node:7008) UnhandledPromiseRejectionWarning: Error: Variable '$where' expected value of type 'DeploymentWhereInput' but got: {"workspace":{"id_in":[null]}}. Reason: 'workspace.id_in[0]' String or Int value expected (line 1, column 8):
query ($where: DeploymentWhereInput) {
       ^
    at BatchedGraphQLClient.<anonymous> (/houston/node_modules/http-link-dataloader/dist/src/BatchedGraphQLClient.js:77:35)
    at step (/houston/node_modules/http-link-dataloader/dist/src/BatchedGraphQLClient.js:40:23)
    at Object.next (/houston/node_modules/http-link-dataloader/dist/src/BatchedGraphQLClient.js:21:53)
    at fulfilled (/houston/node_modules/http-link-dataloader/dist/src/BatchedGraphQLClient.js:12:58)
    at process._tickCallback (internal/process/next_tick.js:68:7)
(node:7008) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 5)
```

**Comment by ryw** [comment]
@petedejoy do you know anything about this area? Last bug I think for v0.10

**Comment by petedejoy** [comment]
I don't, but this looks like we have an error in one of our helper functions or resolvers. @ashb - any thoughts on where this is happening in houston? Maybe the auth directive?

**Comment by ryw** [comment]
Looks like @schnie wrote that code so maybe he can resolve

**Comment by ashb** [comment]
I haven't managed to dig in to it yet. My current guess is that the  service-token to workspace mapping/lookup is failing. In there, or somewhere in `src/routes/v1/registry/authorization/handler.js`

**Comment by ashb** [comment]
Ah. debugged a bit. This error is specific to _deployment_ service accounts. Workspace ones work fine.

**Comment by ashb** [comment]
Fixed by astronomer/orbit-ui#138
